charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -162,17 +174,53 @@ allowedMetrics:
   - process_max_fds
   - process_open_fds
   machineControllerManager:
-  - process_max_fds
-  - process_open_fds
+  - mcm_cloud_api_requests_failed_total
+  - mcm_cloud_api_requests_total
   - mcm_machine_controller_frozen
+  - mcm_machine_current_status_phase
+  - mcm_machine_deployment_failed_machines
```

Comment:
```suggestion
```
The MCM doesn't initialize this variable, hence maybe not be always available. This could cause a problem with the Grafana dashboards if I had understood it right. 

Something to be fixed at the MCM before exposing it here. So we will have to avoid exposing this metric for now right? Refer - https://github.com/gardener/machine-controller-manager/issues/211#issuecomment-486078446.

Reply:
We do not use this metric in the dashboard and afaik not for the alters. If the metrics is not needed I would remove the metric from the whitelist. WDYT?

Reply:
Okay, sure we can remove it for now. And if and when required later, we could re-introduce it.

Reply:
I think that if this is not hurting we could leave it as it is. It gives us information about failing machine-deployments which should help.

Reply:
No it is not hurting we can keep it.